cluster/plugin/local/main.go

```diff
@@ -40,9 +40,11 @@ func initClient(cmd *cobra.Command, args []string) (err error) {
 }
 
 func create(cmd *cobra.Command, args []string) {
-	// docker swarn init --advertise-addr $interface
 	ctx := context.Background()
 
+	if err := plugin.CheckPrerequisites(); err != nil {
```

Comment:
the check on map count should be done only if the log stack is enabled.
There's no need to kill the deployment if the operator is not deploying Elasticsearch.

Reply:
Yes, good call